### PR TITLE
Admin functions for the first iteration - fixes #23

### LINKED ISSUE #23: Implement admin functions for first iteration
Implement the different admin functions described in the "Protocole de communication Client - Serveur" document.

### PR DESCRIPTION
**Pull request by azixus**
The following admin functions are currently handled : 
- newDebate : create a new debate with the specified title and description
- newQuestion : add a question to a debate
- getDebates : get the list of debates

fixes #23

### CODE REVIEW

File: src/namespace/adminnamespace.js [hunk 1]
```diff
@@ -26,16 +26,97 @@ export class AdminNamespace extends CustomNamespace {
...
+
+                // TODO: Only return debates available for this user
+
+                let debates = [];
+                for (const [key, debate] of this.activeDebates.entries()) {
+                    debates.push({
+                        debateId: debate.debateID,
+                        title: debate.title,
+                        description: debate.description
+                    })
```
Comment (ghost) [inline src/namespace/adminnamespace.js]: End of instruction char ';' is missing.
```suggestion
                    });
```

File: test/debate.js [hunk 2]
```diff
@@ -90,6 +99,236 @@ describe('Debate test', () => {
...
+            debate = debateManager.nspAdmin.getActiveDebate(id);
+
+            client = io.connect(`http://localhost:${SocketConfig.SOCKET_PORT}${SocketConfig.DEBATE_NAMESPACE_PREFIX}${id}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+            });
+
+            client.on('connect', () => {
+                done();
+            })
```
Comment (ghost) [inline test/debate.js]: End of instruction char ';' is missing.
```suggestion
            });
```

File: test/debate.js [hunk 3]
```diff
@@ -90,6 +99,236 @@ describe('Debate test', () => {
...
+        });
+
+        describe('getQuestions', () => {
+            it('no questions', (done) => {
+                client.emit('getQuestions', (questions) => {
+                    questions.length.should.equal(0);
+                    done();
+                })
```
Comment (ghost) [inline test/debate.js]: End of instruction char ';' is missing.
```suggestion
                });
```

File: test/debate.js [hunk 4]
```diff
@@ -90,6 +99,236 @@ describe('Debate test', () => {
...
+            debate = debateManager.nspAdmin.getActiveDebate(id);
+
+            client = io.connect(`http://localhost:${SocketConfig.SOCKET_PORT}${SocketConfig.DEBATE_NAMESPACE_PREFIX}${id}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+            });
+
+            client.on('connect', () => {
+                done();
+            })
```
Comment (ghost) [inline test/debate.js]: End of instruction char ';' is missing.
```suggestion
            });
```

**Review by ghost — CHANGES_REQUESTED** [review]
A few end of instruction char ';' are missing but everything else is good.

**Comment by azixus** [comment]
Requested changes fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**